Makefile

```diff
@@ -1,9 +1,52 @@
+VIRTUAL_ENV ?= venv
+PIP=$(VIRTUAL_ENV)/bin/pip
+PYTHON=$(VIRTUAL_ENV)/bin/python
+PYTHON_MAJOR_VERSION=2
+PYTHON_MINOR_VERSION=7
+SYSTEM_DEPENDENCIES= \
+    gir1.2-notify-0.7 \
+    libpython$(PYTHON_VERSION)-dev \
+    python$(PYTHON_VERSION) \
+    python$(PYTHON_VERSION)-dev \
+    python-aptdaemon \
+    python-aptdaemon.gtk3widgets \
+    python-gi \
+    virtualenv
+PYTHON_VERSION=$(PYTHON_MAJOR_VERSION).$(PYTHON_MINOR_VERSION)
+PYTHON_MAJOR_MINOR=$(PYTHON_MAJOR_VERSION)$(PYTHON_MINOR_VERSION)
+PYTHON_WITH_VERSION=python$(PYTHON_VERSION)
+DOCKER_IMAGE=gerardpuig/ubuntu-cleaner
+DOCKER_VOLUME=/tmp/.X11-unix:/tmp/.X11-unix
+
+all: virtualenv
+
+system_dependencies:
+	apt install --yes --no-install-recommends $(SYSTEM_DEPENDENCIES)
+
+$(VIRTUAL_ENV):
+	virtualenv --python=$(PYTHON_WITH_VERSION) --system-site-packages $(VIRTUAL_ENV)
```

Comment:
`--system-site-packages` because we want to pick up few system wide Python packages such as `python-aptdaemon`, `python-gi`. Maybe we can find a way to isolate furthermore in the env in the future

Reply:
indeed, this will be changed in the next version since it should be python 3 compliance, for now looks good!